client/src/components/tabs.vue

```diff
@@ -59,6 +60,13 @@ export default class Tabs extends Vue {
 </script>
 
 <style lang="sass" scoped>
+#tabs-landing-comp
+  max-width: 1264px
+
+.tab-items
+  border-bottom-left-radius: 6px !important
+  border-bottom-right-radius: 6px !important
```

Comment:
https://vuetifyjs.com/en/styles/border-radius/ ?

Reply:
Cool! They have a class for everything. Will update this!

Reply:
I was going to say, "the classes are already there; let's use them".

But I now see that main.ts does NOT import the Vuetify CSS. Maybe it should? Future??? Eg,
```
import 'vuetify/dist/vuetify.min.css'
```

Reply:
Noted! indeed it appears with these helpers we might need to import the CSS.